db/migrate/20210629104700_populate_participant_profiles.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class PopulateParticipantProfiles < ActiveRecord::Migration[6.1]
```

Comment:
How long will this take to run on 2000 participants? I believe all migrations need to run in <60s for deploy to succeed